src/pybind/mgr/alerts/module.py

```diff
@@ -180,8 +180,8 @@ def serve(self):
                     self._send_alert(new_status, diff)
                 last_status = new_status
 
-            self.log.debug('Sleeping for %d seconds', self.interval)
-            ret = self.event.wait(self.interval)
+            self.log.debug('Sleeping for %s seconds', self.interval)
```

Comment:
`self.interval` should be a float actually, is 

> mgr/alerts: "MGR_MODULE_ERROR: Module 'alerts' has failed: %d format: a number is required, not str"

reproducible ? if yes, how?

Reply:
I have not reproduced this,  no.

Reply:
https://rook-io.slack.com/archives/C46Q5UC05/p1590494007085100

Reply:
seems i need a `@rook.io` mail address for joining this workspace. =(

Reply:
in any case, imo `%s` is the safest variant, as it supports strings, floats and ints. Makes little sense to force a type in the debug statement.

Reply:
agreed. but i don't want to cover a bug as settings are supposed to be converted to the type as advertised by the "type" fields in the corresponding OPTIONS element.

Reply:
agreed. My idea is that this PR creates a better error message. At lease prints a debug message containing the possibly invalid value.